server/routes/api/subscriptions.ts

```diff
@@ -0,0 +1,220 @@
+import invariant from "invariant";
+import Router from "koa-router";
+import subscriptionCreator from "@server/commands/subscriptionCreator";
+import { sequelize } from "@server/database/sequelize";
+import auth from "@server/middlewares/authentication";
+import { Subscription, Event, Document } from "@server/models";
+import { authorize } from "@server/policies";
+import { presentSubscription } from "@server/presenters";
+import { SubscriptionEvent } from "@server/types";
+import { assertIn, assertPresent, assertUuid } from "@server/validation";
+import pagination from "./middlewares/pagination";
+
+const router = new Router();
+
+router.post(
+  "subscriptions.list",
+
+  auth(),
+
+  pagination(),
+
+  async (ctx) => {
+    const { user } = ctx.state;
+
+    const { documentId, event } = ctx.body;
+
+    assertPresent(documentId, "documentId is required");
```

Comment:
```suggestion
    assertUuid(documentId, "documentId is required");
```
Need to assert the UUID format or junk strings can get passed to db queries